cpp/include/Ice/InputStream.h

```diff
@@ -350,124 +348,19 @@ namespace Ice
          *
          * @return The encoding version used by the encapsulation.
          */
-        const EncodingVersion& startEncapsulation()
-        {
-            Encaps* oldEncaps = _currentEncaps;
```

Comment:
Moved some functions out of line. Having them inline created dependencies issues.